tern/utils/oci_image.py

```diff
@@ -0,0 +1,32 @@
+
```

Comment:
I think this file should be under: `tern/analyze/oci/helpers.py`

Reply:
I updated the code to add modularity similar to the docker.